### PR TITLE
Update sd-software-installer.md

### PR DESCRIPTION
**Pull request by kkmattil**
Lay out fixed.

### CODE REVIEW

File: docs/data/sensitive-data/tutorials/sd-software-installer.md [hunk 1]
```diff
@@ -67,8 +67,10 @@ Notes about the available tools. Note that this list may be out-dated:
 ### Imaging and vidos
```
Comment (ainoc) [inline docs/data/sensitive-data/tutorials/sd-software-installer.md:67]: vidos -> videos

**Review by ainoc — COMMENTED** [review]
*  This tool works only in Ubuntu22 based VMs.
->    *  This tool works only in Ubuntu22 based virtual desktops.

File: docs/data/sensitive-data/tutorials/sd-software-installer.md (hunk 1 above)
Comment (kkmattil) [reply docs/data/sensitive-data/tutorials/sd-software-installer.md]: Korjattu.

**Comment by rkronberg** [comment]
Is this PR ok? Regarding the virtual machine vs. virtual desktop comment, I believe they are used synonymously, at least in the context of the SD Desktop (this is mentioned in https://docs.csc.fi/data/sensitive-data/sd_desktop/). The currently edited page also only uses virtual machine term, so maybe continue with that? @kkmattil @ainoc

**Review by rkronberg — APPROVED** [review]

**Pull request merged**